provisions/roles/openshift/tasks/config.yml

```diff
@@ -9,7 +9,7 @@
   replace: >
     dest=/etc/sysconfig/docker
     regexp="^#*\s*ADD_REGISTRY=.*"
-    replace='ADD_REGISTRY="--add-registry {{ public_registry }}:5000"'
+    replace='ADD_REGISTRY="--add-registry {{ public_registry }}:5000 --add-registry registry.centos.org"'
```

Comment:
>Images should be pulled in following sequence
> internal registry
> registry.centos.org
> docker.io

so {{public_registry}} here is not really public registry but internal registry ?